PC/getpathp.c

```diff
@@ -245,7 +245,8 @@ static void
 join(wchar_t *buffer, const wchar_t *stuff)
 {
     if (_PathCchCombineEx_Initialized == 0) {
-        HMODULE pathapi = LoadLibraryW(L"api-ms-win-core-path-l1-1-0.dll");
+        HMODULE pathapi = LoadLibraryExW(L"api-ms-win-core-path-l1-1-0.dll", NULL,
```

Comment:
Do we need to check that `LoadLibraryExW` exists on the machine? It looks like KB2533623 
 needs to be installed: https://docs.microsoft.com/en-us/windows/win32/api/libloaderapi/nf-libloaderapi-loadlibraryexw

Reply:
The check is in the installer (I backported it for 3.7).

Considering Windows 7 is EOL, I don't think it's unreasonable to require the update.

Reply:
Ah cool, just noticed that. Thanks!